core/native/native_bind.gen.h

```diff
@@ -31,6 +31,33 @@
 namespace carbon {
 
 
+template<typename T> struct is_shared_ptr : std::false_type {};
```

Comment:
TODO: since the generated file is too large in, and it could be generated. add this to scons Builder and generate each time the generator updates